server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -356,6 +356,12 @@ public class ShuffleServerConf extends RssBaseConf {
       .withDescription("The memory usage limit ratio for huge partition, it will only triggered when partition's "
           + "size exceeds the threshold of '" + HUGE_PARTITION_SIZE_THRESHOLD.key() + "'");
 
+  public static final ConfigOption<Long> SERVER_DECOMMISSION_CHECK_INTERVAL = ConfigOptions
+      .key("rss.server.decommission.check.interval")
+      .longType()
+      .defaultValue(60 * 1000L)
```

Comment:
Do we need `checkValue`?

Reply:
Do you mean check if the value is valid?

Reply:
> Do you mean check if the value is valid?

Yes.